Sources/App/Views/Search/SearchShow+Model.swift

```diff
@@ -68,3 +68,16 @@ enum SearchShow {
     }
 
 }
+
+extension String {
+    func filteredForDisplay() -> String {
+        // This does not strip HTML tags as it's possible people would want to search for them.
+        // Instead, it changes angle brackets and quotes and when combined with our SQL sanitisation does everything we need.
+        self.replacingOccurrences(of: ">", with: "&gt;")
+            .replacingOccurrences(of: "<", with: "&lt;")
+            .replacingOccurrences(of: "'", with: "&apos;")
+            .replacingOccurrences(of: "\"", with: "&quot;")
+        // Also, the `term` coming back out of the view model may contain backslash escape characters which we should not display.
```

Comment:
This is actually an unrelated fix. On the current site, if you search for anything that gets sanitised as part of `Search.sanitize`, the escape character was being echoed back to the output in HTML.

Unrelated fix, but worth doing at the same time!